src/client/ui/styles.less

```diff
@@ -566,18 +566,29 @@
         }
 
         .selector-inspector-panel {
-            display: flex;
             box-sizing: border-box;
             width: 100%;
             left: 0;
-            padding: 2px;
+            padding: 2px 76px 2px 2px;
```

Comment:
I think that here, we do not need add extra padding on the right side.
I belive, that in the `show inspector panel` scenario the `Hide` button position should not be `absolute`. I think we need to add the `Hide` button after inspector panel, but not before it. Then after click it should change position to absolute and go top.